src/Conference/UpdateConference.jsx

```diff
@@ -0,0 +1,153 @@
+import React, { useState, useEffect } from "react";
+import Input from "../Input";
+import styled from "styled-components";
+import { useHistory, useParams } from "react-router-dom";
+import { updateConferenceById, getConferenceById } from "../api";
+
+const StyledConferenceForm = styled.div`
+  height: 400px;
+  padding: 20px;
+`;
+
+const StyledCardHeading = styled.h3`
+  background: #dbd8db;
+  width: 100%;
+  height: 55px;
+  font-size: 30px;
+  font-weight: bold;
+  text-align: center;
+  vertical-align: middle;
+`;
+
+const StyledCard = styled.div`
+  border-style: solid;
+  border-color: black;
+  border-radius: 25px;
+  background: #dbd8db;
+  width: 65%;
+  height: 370px;
+  transition: 0.3s;
+  :hover {
+    box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+  }
+  vertical-align: middle;
+  margin: 0 auto;
+`;
+
+const UpdateForm = () => {
+  let history = useHistory();
+  const { id } = useParams();
+
+  const [conference, setConference] = useState({
+    name: "",
+    dateTime: "",
+    city: "",
+    description: "",
+    topic: ""
+  });
+
+  const fetchData = async conferenceId => {
+    try {
+      const response = await getConferenceById(conferenceId);
+      setConference({
+        name: response.name,
+        dateTime: response.dateTime.substring(0, response.dateTime.length - 1),
+        city: response.city,
+        description: response.description,
+        topic: response.topic
+      });
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  useEffect(() => {
+    if (id) {
+      fetchData(id);
+    }
+  }, [id]);
+
+  const handleChange = event => {
+    const { name, value } = event.target;
+    setConference({ ...conference, [name]: value });
+  };
+
+  const submitForm = async () => {
+    const newConference = {
+      ...conference,
+      dateTime: `${conference.dateTime}Z`
+    };
+    try {
+      await updateConferenceById(id, newConference);
+      history.push(`/${id}`);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  return (
+    <StyledConferenceForm>
+      <StyledCard>
+        <StyledCardHeading>Edit a conference:</StyledCardHeading>
+        <form
+          onSubmit={ev => {
+            ev.preventDefault();
+            submitForm();
+          }}
+        >
+          <Input
+            label="Conference Name: "
+            type="text"
+            name="name"
+            value={conference.name}
+            onChange={handleChange}
+            required
+            maxLength="80"
+          />
+          <br />
```

Comment:
Generally probably shouldn't use <br>, either wrap Input in block element like <p> or use CSS to break the line